docs/admin/logging/gcplogs.md

```diff
@@ -0,0 +1,63 @@
+<!--[metadata]>
++++
+aliases = ["/engine/reference/logging/gcplogs/"]
+title = "Google Cloud Logging driver"
+description = "Describes how to use the Google Cloud Logging driver."
+keywords = ["gcplogs, google, docker, logging, driver"]
+[menu.main]
+parent = "smn_logging"
+weight = 2
++++
+<![end-metadata]-->
+
+# Google Cloud Logging driver
+
+The Google Cloud Logging driver sends container logs to [Google Cloud
+Logging](https://cloud.google.com/logging/docs/).
```

Comment:
Can you make this a HTML link with a `target="_blank"`? Overall, we try to open external links in a new tab/window so that readers can more easily continue reading the docs where they left off. i.e.;

```Markdown
<a href="https://cloud.google.com/logging/docs/" target="_blank">Google Cloud Logging</a>
```

(yeah, it's not as nice as a plain markdown link, but it does the job :smile:)